Libraries/rocSPARSE/level_3/gebsrmm/.gitignore

```diff
@@ -0,0 +1 @@
+rocsparse_gebsrmm
```

Comment:
Why should it be ignored?

Reply:
This is only the executable generated after compiling, we have a .gitignore inside each example's folder so that we don't actually push the executables to the repo